frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -68,6 +68,7 @@
         vm.allowedSubmissionFileTypes = [];
         vm.currentPhaseAllowedSubmissionFileTypes = '';
         vm.defaultSubmissionMetaAttributes = [];
+        vm.submissionMetaDataTemp = null;
```

Comment:
@burnerlee can you rename the variable to something like `vm.currentSubmissionMetaData`